Documentation/network/servicemesh/istio.rst

```diff
@@ -36,35 +36,44 @@ You can confirm the result with the following command:
     $ kubectl get configmaps -n kube-system cilium-config -oyaml | grep bpf-lb-sock-hostns
     bpf-lb-sock-hostns-only: "true"
 
+
+To ensure that Cilium does not interfere with other CNI plugins on the node, it is important to set the
+``cni-exclusive`` parameter in the Cilium ConfigMap to ``false``. This can be achieved by using the
+``--set`` flag with the ``cni.exclusive`` Helm value set to ``false``.
+You can confirm the result with the following command:
+
+.. code-block:: shell-session
+
+    $ kubectl get configmaps -n kube-system cilium-config -oyaml | grep cni-exclusive
+    cni-exclusive: "false"
+
```

Comment:
I'm interested in getting input from others on the preferred UX for changing the configmap. Instead, should we lead users to [cilium CLI](https://docs.cilium.io/en/stable/configuration/), e.g. `cilium config set`?

Reply:
Probably also worth discussing managed k8s (ie GKE, DOKS, etc) where the CLI may not be a good approach/overridden by the provider.

Reply:
From my understanding, the preferred approach is anything helm-based, ie `cilium install` or `helm upgrade --install`. This ties into the installation methods that we have in the Getting Started guide.

Reply:
both are probably needed, but for now I just used the existing convention.